### PR TITLE
Standardize on App Veyor

### PR DESCRIPTION
**Pull request by UebelAndre**
I'm unsure if Travis is preferable over App Veyor but I don't think using both is the best idea. This PR takes all the functionality that was run in Travis and runs it in App Veyor.

### CODE REVIEW

**Comment by UebelAndre** [comment]
@richgel999 This PR is ready for review

**Comment by UebelAndre** [comment]
@madebr What do you think about this PR?

File: appveyor.yml [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
+  - Ubuntu2004
+  - Visual Studio 2019
 
 configuration: Release
 
 environment:
   APPVEYOR_YML_DISABLE_PS_LINUX: true
+
+install:
+- sh: |
```
Comment (madebr) [inline appveyor.yml:14]: Do the `sh` commands only run on Linux and Macos?

File: webgl/transcoder/build/basis_loader.js [hunk 2]
```diff
@@ -1,494 +1,494 @@
-/*
```
Comment (madebr) [inline webgl/transcoder/build/basis_loader.js:1]: The changes to the *.MD files and this javascript, were these don to please `./format.sh`?

File: appveyor.yml (hunk 1 above)
Comment (UebelAndre) [reply appveyor.yml:14]: Yes (at least by default anyway). There may be a similar variable to `APPVEYOR_YML_DISABLE_PS_LINUX` to enable bash on windows. But yeah, this doesn't run on windows.

File: webgl/transcoder/build/basis_loader.js (hunk 2 above)
Comment (UebelAndre) [reply webgl/transcoder/build/basis_loader.js:1]: All I did was run `./format.sh` and commit the results. So yeah.

Comment (UebelAndre) [reply webgl/transcoder/build/basis_loader.js:1]: If I recall correctly, this is fixing the line endings of these files.

**Comment by madebr** [comment]
The changes look good to me.
It depends on how @richgel999 feels about moving away from travis-ci.
travis-ci also support Windows, Linux and Macos, and supports multiple parallel jobs.

But you can't support all ci services.
So you get a :+1:  from me.

**Comment by UebelAndre** [comment]
Yeah, I only added this to Appveyor because that's what ran when I opened my first pull request. Saw no sign of travis. I think there should only be one CI system and if it should be travis then there should have been no reason to merge #173 

If there's a future plan to move to travis, then I would still say this change is worth merging and when the migration happens, the code base will still have full coverage.

**Comment by UebelAndre** [comment]
@richgel999 Hey, any chance I could get your eyes on this PR? 🙏

**Comment by UebelAndre** [comment]
@richgel999 Hey, just pinging this PR again 😅

**Comment by UebelAndre** [comment]
@richgel999 hey, just pinging this PR again 😅

**Comment by richgel999** [comment]
Got it - thanks a lot!

**Pull request merged**